AnnoDesigner/app.manifest

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
```

Comment:
maybe should be adjusted

Reply:
That was just the default way the file was created when I did "Add new item" from visual studio.

Reply:
Ah - nevermind - just seen what you saw...

Reply:
Updated